en/middleware/mavlink.md

```diff
@@ -375,6 +392,13 @@ In this case the [BatteryStatus](../msg_docs/BatteryStatus.md) uORB topic alread
 uORB::Publication<battery_status_s> _battery_pub{ORB_ID(battery_status)};
 ```
 
+This creates a publication to a single uORB topic instance, which by default will be the _first_ instance.
+
+:::note
+This implementation won't work on multi-battery systems, because several batteries might be publishing data to the first instance of the topic, and there is no way to differentiate them.
+To support multiple batteries we'd need to use `PublicationMulti` and map the MAVLink message instance IDs to specific uORB topic instances.
+:::
```

Comment:
Explanation of why this implementation is flawed. I think worth saying, and pointing out PublicationMulti